test/test_other.py

```diff
@@ -12423,3 +12423,9 @@ def test_recursive_cache_lock(self):
     with shared.Cache.lock('testing'):
       err = self.run_process([EMBUILDER, 'build', 'libc', '--force'], stderr=PIPE, check=False).stderr
     self.assertContained('AssertionError: attempt to lock the cache while a parent process is holding the lock', err)
+
+  # Tests building the AudioWorklet demo (does not run.. see test interactive.test_audio_worklet for the running version)
+  @also_with_minimal_runtime
+  def test_audio_worklet(self):
+    self.run_process([EMCC, test_file('webaudio/audioworklet.c'), '-sAUDIO_WORKLET', '-sWASM_WORKERS'])
+    self.run_process([EMCC, test_file('webaudio/audioworklet.c'), '-sAUDIO_WORKLET', '-sWASM_WORKERS', '--closure', '1', '-Oz'])
```

Comment:
Maybe this could be in `test_browser` even if we don't have the interactive part? That would test both compilation and that no error happens during runtime.

Reply:
Yeah, that's a great idea